pkg/distributor/distributor.go

```diff
@@ -322,8 +300,9 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 			Help:      "The configured replication factor.",
 		}),
 		latestSeenSampleTimestampPerUser: promauto.With(reg).NewGaugeVec(prometheus.GaugeOpts{
-			Name: "cortex_distributor_latest_seen_sample_timestamp_seconds",
-			Help: "Unix timestamp of latest received sample per user.",
+			Namespace: "cortex",
+			Name:      "distributor_latest_seen_sample_timestamp_seconds",
```

Comment:
Nit: please don't introduce using `Namespace`, but use `Name` only. It makes it easier to find the metric name in the codebase.

Reply:
OK. I was updating this one because the rest of the metrics in the file are `Namespace` + `Name` style. Do you think I should update them all to use `Name` only? Or just revert this instance?